src/uniref/config/UniRefColumnConfiguration.tsx

```diff
@@ -137,19 +144,33 @@ UniRefColumnConfiguration.set(UniRefColumn.types, {
 
 UniRefColumnConfiguration.set(UniRefColumn.members, {
   label: 'Members',
-  render: ({ members }) => (
-    <ExpandableList descriptionString="members" displayNumberOfHiddenItems>
-      {members?.map((member) => (
-        <Link
-          key={member}
-          to={generatePath(LocationToPath[Location.UniProtKBEntry], {
-            accession: member,
-          })}
-        >
-          {member}
-        </Link>
+  render: ({ members, memberCount, id }) => (
+    <ul className="no-bullet">
+      {members?.slice(0, CUT_OFF).map((member) => (
+        <li key={member}>
+          <Link
+            to={getEntryPath(
+              member.startsWith('UPI')
+                ? Namespace.uniparc
+                : Namespace.uniprotkb,
+              member
+            )}
+          >
+            {member}
+          </Link>
+        </li>
       ))}
-    </ExpandableList>
+      {members.length > CUT_OFF && (
+        <Button
+          element={Link}
+          variant="tertiary"
+          to={getEntryPath(Namespace.uniref, id)}
+        >
+          {memberCount - CUT_OFF} more member
```

Comment:
For consistancy, the "M" of more should be capitalised

Reply:
Not here since there is a number in front of it